packages/discord.js/src/structures/Message.js

```diff
@@ -329,7 +329,8 @@ class Message extends Base {
      * @typedef {Object} MessageInteraction
      * @property {Snowflake} id The interaction's id
      * @property {InteractionType} type The type of the interaction
-     * @property {string} commandName The name of the interaction's application command
+     * @property {string} commandName The name of the interaction's application command,
+     * including subcommands and subcommand groups
```

Comment:
I think this would make it clearer, it shows only one, not all of them.
```suggestion
     * @property {string} commandName The name of the interaction's application command,
     * as well as the subcommand and subcommand group, where applicable
```